runtime/doc/options.txt

```diff
@@ -7594,9 +7438,10 @@ A jump table for the options with a short description can be found at |Q_op|.
 	but the Windows version of Vim can source unix format scripts.
 
 				*'viminfo'* *'vi'* *E526* *E527* *E528*
-'viminfo' 'vi'		string	(Vi default: "", Vim default for MS-DOS,
-				   Windows: '100,<50,s10,h,rA:,rB:,
-				   for others: '100,<50,s10,h)
+'viminfo' 'vi'		string	(Vim default for
+				   Win32: "'100,<50,s10,h,rA:,rB:",
+				   others: "'100,<50,s10,h",
```

Comment:
But this option's defaults are pretty confusing to read, especially since comma is also part of the option, yet it's also used as punctuation in the documentation. I think alignment could be used to make it more readable (removing `"` and terminating `,`):

```
'viminfo' 'vi'         string  (Vim default for
				   Win32:  '100,<50,s10,h,rA:,rB:
				  others:  '100,<50,s10,h
				Vi default: "")

```